Resources/Textures/Nuclear14/Objects/Weapons/Guns/Rifles/Chinese Rifle.rsi/meta.json

```diff
@@ -0,0 +1,28 @@
+{
+    "version": 1,
+    "license": "CC-BY-SA-3.0",
+    "copyright": "what im supposed to put in here? i made it myself",
```

Comment:
"Made by @PatoGrone for Nuclear14." Up to you if you want to add your discord handle too although they're changing soon.

Reply:
i forgot about those xd